CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Contribution guidelines
+
++ [Before adding a recipe](#before-adding-a-recipe)
+  + [Labels](#labels)
++ [Recipe Structure](#recipe-structure)
+  + [Badges](#badges)
++ [Pull requests](pull-requests)
++ [Instructions for maintainers](#instructions-for-maintainers)
+
+## Before adding a recipe
```

Comment:
Would change this to something that would include tweaking a recipe or adding an ingredient that was missing.